.travis.yml

```diff
@@ -32,32 +32,21 @@ branches:
 before_install:
   # Fixes Composer exhausting the memory when using PHP 5.6.
   - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - nvm install 6.11.0
-  - nvm use 6.11.0
-  # Repo for Yarn
-  - sudo apt-key adv --keyserver pgp.mit.edu --recv D101F7899D41F3C3
-  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-  - sudo apt-get update -qq
-  - sudo apt-get install -y -qq yarn
   - mkdir $PROJECT_BASE_PATH
+  # Load .env and .local.env
+  - source ./scripts/ci/travis/load_env.sh
 
 install:
-  - yarn install
   - composer run-script install-contenta $PROJECT_BASE_PATH --timeout=0
-
+  - composer run-script ci:helper setup_anonymous_user "$PROJECT_BASE_PATH" --timeout=0
```

Comment:
I still strongly believe we should have a composer.json script to make it as easy as possible to run the tests aka. ```composer run-script tests````